images/win/scripts/Tests/Java.Tests.ps1

```diff
@@ -1,5 +1,6 @@
 Describe "Java" {
-    [array]$jdkVersions = (Get-ToolsetContent).java.versions | ForEach-Object { @{Version = $_} }
+    $defaultVendor = (Get-ToolsetContent).java.default_vendor
+    [array]$jdkVersions = ((Get-ToolsetContent).java.vendors | Where-Object {$_.name -eq $defaultVendor}).versions | ForEach-Object { @{Version = $_} }
```

Comment:
I see we get only "Temurin-Hotspot" version. Where about Adopt are we planning to test them?

Reply:
@al-cheb it was concerned above https://github.com/actions/virtual-environments/pull/3985#discussion_r698445503

Reply:
@mikhailkoliada , I mean very simple test just to validate that these versions exist.

Reply:
@mikhailkoliada @maxim-lobanov Do we need to test Java-13 anyway as it's in the PATH?

Reply:
@miketimofeev I think it got changed on this stage https://github.com/actions/virtual-environments/pull/3985#discussion_r698445503